src/api/assets.js

```diff
@@ -101,6 +101,28 @@ class AssetsAPI {
       return json
     }
   }
+
+  static getPreferredFormat = (asset) => {
```

Comment:
Is there a better way to do this? So we're not hardcoding value checks? Or is the order important?

indexof() returns  -1 if there's no index, that could be used in a comparator instead.

Reply:
The order is in decreasing preference for asset type yes! this is a fairly short list at the moment but once we accept more asset types it will matter more. Just in this example, we prefer to render GLTF2 over TILT files as the tilt renderer needs a lot of work. Happy for suggestions on how to do this better!